tensorflow/training/docker/2.7/py3/example/Dockerfile.gpu

```diff
@@ -0,0 +1,8 @@
+# Expecting base image to be the image built by ../cu112/Dockerfile.e3.gpu
+ARG BASE_IMAGE=""
+
+FROM $BASE_IMAGE
+
+# Add any script or repo as required
+RUN git clone -b v0.1 https://github.com/aws-samples/deep-learning-models
```

Comment:
nit: should we update it to the latest release v0.2?

Reply:
@tejaschumbalkar feel free to push a commit to update it

Reply:
Let's stick with v0.1 for now - it looks like some significant changes of v0.2, we can keep this in mind for future releases